lib/logitech_receiver/notifications.py

```diff
@@ -274,4 +274,31 @@ def _process_feature_notification(device, status, n, feature):
 			_log.warn("%s: unknown TOUCH MOUSE %s", device, n)
 		return True
 
+	if feature == _F.DEVICE_FW_VERSION:
+		if (n.address == 0x00) and (n.sub_id == 0x02):
+			if _log.isEnabledFor(_INFO):
+				delta_h, delta_v, wheel = _unpack('<xbhb', n.data[:5])
+				_log.info("%s: MOUSE movement: delta H:%-3d delta V:%-3d wheel: %-3d", device, delta_h, delta_v, wheel)
+			return True
+		else:
+			_log.warn("%s: unknown FW_VERSION %s", device, n)
```

Comment:
The logitech HID driver handles those events. Yet, they arrive to Solaar, via hidraw interface. Without this change, when in debug mode, it prints a series of unknown events, like this one:

<pre>
06:18:06,630  WARNING [ReceiverListener:hidraw1] logitech_receiver.notifications: &lt;PairedDevice(3,404A,Anywhere MX 2)&gt;: unrecognized Notification(3,02,00,0000000000000000000000) for feature DEVICE FW VERSION (index 02)
</pre>

Anyway, I'll send a pull request without the patch fixing this log message handling.